File: Wirtualny świat/java/src/organizm/owca.java

package organizm;
import swiat.swiat;


public class owca extends zwierze {
    public owca(swiat WielkiSwiat, int x, int y)
    {
        super(WielkiSwiat, x, y);
        sila = 2;
	inicjatywa = 4;
        this.nazwa = 'O';
        
    }
   
    @Override
    public void nowy(swiat WielkiSwiat, int x, int y)
    {
        organizm o = new owca(WielkiSwiat, x, y);
    }
    public String nazwa()
    {
        return "Owca";
    }
}

File: Wirtualny świat/C++/ConsoleApplication18/owca.cpp
#include "stdafx.h"
#include "owca.h"


owca::owca(swiat& ref, int x, int y) :zwierze(ref, x, y)
{
	this->sila = 4;
	this->inicjatywa = 4;
	this->nazwa = 'O';
	this->specjalna_kolizja = 0;
}

char owca::rysowanie() {
	return 'O';
}
void owca::nowe(swiat& ref, int x, int y)
{
	organizm *o = new owca(ref, x, y);
	cout << "dodano: " << this->rysowanie() << " na polu: " << x + 1 << ':' << y + 1 << ":\n";
}
owca::~owca()
{
}
